src/prompts/followUp/fiscalYear.ts

```diff
@@ -0,0 +1,13 @@
+const fiscalYear = `
+Extract the company fiscal year. Sometimes companies have broken fiscal year.  Example: 1 apr -> 31 mar means: startMonth = 4, endMonth = 3. Standard is 1 jan -> 31 dec. 
```

Comment:
Will `->` be parsed correctly as an arrow? Or would it be better to write an explicit `to` instead, replacing `->`?

Reply:
It should be fine.